app/javascript/courses/review/components/CoursesReview__Editor.res

```diff
@@ -999,6 +999,20 @@ let updateReviewer = (cb, send, reviewer) => {
   send(ShowGradesEditor)
 }
 
+let pageTitle = (number, submissionDetails) => {
+  let studentOrTeamName = switch SubmissionDetails.teamName(submissionDetails) {
+  | Some(teamName) => teamName
+  | None => {
+      let students = SubmissionDetails.students(submissionDetails)
+      Student.name(students[0])
+    }
+  }
```

Comment:
Submissions can belong to multiple students. Let's map over the student names

Reply:
Oh, I didn't know about this case. I thought a submission will have multiple student names only if they are in team.